.changeset/fifty-yaks-behave.md

```diff
@@ -0,0 +1,6 @@
+---
+'example-app': patch
```

Comment:
Since we don not publish the `example-app`, we don't need to include it in the changeset.